packages/firebase_crashlytics/firebase_crashlytics/example/android/app/build.gradle

```diff
@@ -32,10 +32,9 @@ android {
     }
 
     defaultConfig {
-        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "io.flutter.plugins.firebase.crashlytics.firebasecrashlyticsexample"
+        applicationId 'io.flutter.plugins.firebasecrashlyticsexample'
         minSdkVersion 16
-        targetSdkVersion 27
+        targetSdkVersion 28
```

Comment:
if this is getting bumped , i wonder if it's worth bumping it to 29 even if it doesn't change much/anything

Reply:
We'll be bumping all example apps separately to 30, already tracking this - thanks :)

Reply:
Np, I guess a similar comment applies to the compile SDK version specified for the plugin